src/main/scala/devices/debug/Periphery.scala

```diff
@@ -40,7 +40,7 @@ case object ExportDebug extends Field(DebugAttachParams())
 
 class ClockedAPBBundle(params: APBBundleParameters) extends APBBundle(params) with Clocked
 
-class DebugIO(implicit val p: Parameters) extends ParameterizedBundle()(p) with CanHavePSDTestModeIO {
+class DebugIO(implicit val p: Parameters) extends ParameterizedBundle()(p) {
```

Comment:
`ParameterizedBundle` is now unnecessary if all the parameters of the class are `val`s (as is the case here). So you can just extend `Bundle`